lib/sisimai/bite/email/sendgrid.rb

```diff
@@ -0,0 +1,223 @@
+module Sisimai::Bite::Email
+  module SendGrid
+    # Sisimai::Bite::Email::SendGrid parses a bounce email which created by
+    # SendGrid. Methods in the module are called from only Sisimai::Message.
+    class << self
+    # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/SendGrid.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :'from'        => %r/\AMAILER-DAEMON\z/,
+        :'return-path' => %r/\A[<]apps[@]sendgrid[.]net[>]\z/,
+        :'subject'     => %r/\AUndelivered Mail Returned to Sender\z/,
+      }
+      Re1 = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->